bundler.d/test.rb

```diff
@@ -6,7 +6,6 @@
   gem 'public_suffix', '< 3' if RUBY_VERSION < '2.1'
   gem 'benchmark-ips'
   gem 'ruby-prof'
-  gem 'rubocop', '0.38.0'
```

Comment:
Won't this be an issue for jenkins?

Reply:
I didn't think so, but now that I checked it, yes... In core it's implemented differently, but anyway I'll revert this.

Reply:
Now that it's back in test.rb you probably can't see the full context but it was moved to development.rb. IIRC Jenkins does `bundle install --without development` and not have rubocop installed.